packages/elements/src/color-dialog/index.ts

```diff
@@ -453,8 +457,10 @@ export class ColorDialog extends Dialog {
               id="blueInput"
             ></ef-number-field>
           </div>
-          <div>#&nbsp;:
+          <div>
+            <label id="hexLabel" aria-label="Hex">#&nbsp;:</label>
             <ef-text-field
+              aria-labelledby="hexLabel"
```

Comment:
@goremikins This is the issue I've been looking. Visual label is not the same with screen reader label. My preferred choice would be just to use `aria-label="Hex"`, but standalone `#` would need to be hidden as it would announce as 'number' and cause confusion.

```suggestion
              aria-label="Hex"
```

Or this https://github.com/Refinitiv/refinitiv-ui/commit/1fe31bbf75213299d1bd9e6f0f1be42173ccf96a#diff-0b58f023c0e3636547fab7576bf82f54260fe086feb58dae52308af6f6b32047R461 but I find it a bit redundant.